docs/build-configuration/assets.md

```diff
@@ -6,14 +6,14 @@ permalink: /build/assets/
 redirect_from: /assets/
 published: true
 ---
```

Comment:
Technically you shouldn't have the comment in this sentence.

Reply:
Get out the ruler!